build-system/compile/generate/metadata/bento-runtime-packages.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @fileoverview
+ * These are the packages, and their exports that are included in `bento.js`
+ * Extension `bento-*.js` binaries will use these exports as provided by
+ * `bento.js` from the `BENTO` global.
+ *
+ * We specify each export explicitly by name.
+ * Unlisted imports will be bundled with each binary.
+ */
+
+module.exports = {
+  '#preact': [
+    'createElement',
+    'cloneElement',
+    'render',
+    'hydrate',
+    'Fragment',
+    'createRef',
+    'createContext',
+    'useState',
+    'useRef',
+    'useEffect',
+    'useLayoutEffect',
+    'useContext',
+    'useMemo',
+    'useCallback',
+    'useImperativeHandle',
+  ],
+  '#preact/component': ['ContainWrapper', 'Wrapper'],
+  '#preact/compat': ['forwardRef'],
+  '#preact/base-element': ['PreactBaseElement'],
+};
```

Comment:
maybe i'm misunderstanding, but why wouldn't we centralize things in `'#core/...` or `'#utils/...`?

Reply:
nvm I'm mistaken: #core/dom is bundled in the bento.js bundle already as shown in the  source-map-explorer here: https://github.com/ampproject/amphtml/issues/36899

Reply:
for posterity:

we talked offline and
* my understanding is correct - excluding `#core/...` and `#utils/...` does mean that they get duplicated between bento.js runtime and the extension scripts. See below for screencaps of bento.js and bento-accordion-1.0.js for examples:
* we're still going to exclude '#core/...` and `#utils/...` because we're fine with duplicating - they're small and used very granularly
* we will add `useSlotContext ` from `preact/slotcontext` to centralize the core/context so it's not duplicated across extension bundles accidentally

bento.js bundle breakdown
![image](https://user-images.githubusercontent.com/6854875/142462366-fd65fef6-315a-4191-8930-724308e6a2fb.png)

bento-accordion-1.0.js bundle breakdown
![image](https://user-images.githubusercontent.com/6854875/142463072-3a9ecef4-39d4-4b3b-9991-b5e1263ff9a1.png)

Reply:
Fixed by including whole modules, and listing `core/context`, `preact/context` and `preact/slot`